viewing_party/main.py

```diff
@@ -0,0 +1,251 @@
+#WAVE ONE
+
+def create_movie(title, genre, rating):  
+    movie_dict = {} 
+    if title and genre and rating:
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+        return movie_dict
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie) 
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    #This represents that the user has a watchlist and\
+    #a list of watched movies
+    for movie_dict in user_data["watchlist"]:
+        if movie_dict["title"] == title:
+            user_data["watchlist"].remove(movie_dict)
+            user_data["watched"].append(movie_dict)
+            return user_data
+    else:
+        return user_data
+
+#WAVE TWO
+
+def get_watched_avg_rating(user_data):
+    # the value of user_data will be a dictionary with a\
+    #a "watched" list of movie dictionaries
+    #This represents that the user has a list of watched\
+    #movies
+    avg_rating = 0.0
+    count =0
+    rating_sum = 0.0
+    for movie_dict in user_data["watched"]:
+        for key,value in movie_dict.items():
+            if key == "rating":
+                rating_sum += value
+                count += 1
```

Comment:
Because `movie_dict` is a dictionary you can take advantage of that and use the key-value pairs a bit.  This avoids the need for a loop.  That's the awesome thing about dictionaries, if you know the key you want, you *don't* need to loop through the dictionary to find it, you can just use the key and get the value. 

```suggestion
        rating_sum += movie_dict["rating"]
        count += 1
```